### PR TITLE
Fix TWP38

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#39 by:
* Accomplishing with the checklist.
* Adding brython module in the final exercise for the `eval` function to work.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

### CODE REVIEW

File: _sources/lectures/TWP38.rst [hunk 1]
```diff
@@ -56,28 +56,26 @@ Lenguajes de programación
...
++ Los intérpretes analizan y ejecutan el programa instrucción por
+  instrucción del lenguaje de máquina
 
 
 Python
 ======
 
 
-
 + El signo ">>>" indica que Python está "listo" para recibir
```
Comment (y-martinez) [inline _sources/lectures/TWP38.rst:69]: Maybe this should remove it until we can add the Python interpreter here if is possible.

File: _sources/lectures/TWP38.rst [hunk 2]
```diff
@@ -263,13 +261,13 @@ Resumen
...
-+ Los algoritmos son programas (software) que determinan cuáles son los
-   la computadora (hardware) hará
++ Los algoritmos son programas (software) que determinan que hará
+  la computadora (hardware).
 
-+ El proceso de creación de software se llama programación
++ El proceso de creación de software se llama programación.
 
 
 Recuerdo
```
Comment (y-martinez) [inline _sources/lectures/TWP38.rst:273]: Recuerda o Recuerde

File: _sources/lectures/TWP38.rst [hunk 3]
```diff
@@ -283,10 +281,9 @@ Recuerdo
 + Python es un lenguaje interpretado de alto nivel
 
 + El intérprete de Python convierte los comandos instrucción por instrucción
-   para lenguaje máquina
+  para lenguaje máquina
```
Comment (y-martinez) [inline _sources/lectures/TWP38.rst:284]:
```suggestion
  a lenguaje máquina
```

File: _sources/lectures/TWP38.rst [hunk 4]
```diff
@@ -152,94 +150,95 @@ Python
...
-.. activecode:: Example13_5
-   :nocodelens:
-   :stdin:
+.. activecode:: ac_l38_1
+    :language: python3
+    :python3_interpreter: brython
 
-    print('Este programa ilustra el comportamiento caótico.')
-    x = eval(input('Ingrese un número entre 0 y 1:'))
+    print("Este programa ilustra el comportamiento caótico.")
```
Comment (y-martinez) [inline _sources/lectures/TWP38.rst:164]: This example doesn't show anything but in the browser's console appears the output.

**Review by y-martinez — CHANGES_REQUESTED** [review]

File: _sources/lectures/TWP38.rst (hunk 4 above)
Comment (LeoCumpli21) [reply _sources/lectures/TWP38.rst:164]: It actually does display in the browser's console. Locally, It displays it just fine. 

caos

File: _sources/lectures/TWP38.rst [hunk 5]
```diff
@@ -56,28 +56,26 @@ Lenguajes de programación
...
+  instrucción del lenguaje de máquina
 
 
 Python
 ======
 
 
-
 + El signo ">>>" indica que Python está "listo" para recibir
   comandos
```
Comment (LeoCumpli21) [inline _sources/lectures/TWP38.rst:70]:
```suggestion
```

File: _sources/lectures/TWP38.rst (hunk 2 above)
Comment (LeoCumpli21) [inline _sources/lectures/TWP38.rst:273]:
```suggestion
Recuerde
```

**Review by y-martinez — APPROVED** [review]
LGTM 👍🏼

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

**Pull request merged**